iothub/service/src/ClientApiVersionHelper.cs

```diff
@@ -1,14 +1,34 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+
 namespace Microsoft.Azure.Devices
 {
-    class ClientApiVersionHelper
+    /// <summary>
+    /// Holds the API version numbers required in data-plane calls to the service
+    /// </summary>
+    internal class ClientApiVersionHelper
     {
-        const string ApiVersionQueryPrefix = "api-version=";
-        const string ApiVersionGA = "2016-02-03";
-        const string ApiVersionLatest = "2019-03-30";
-        public const string ApiVersionQueryString = ApiVersionQueryPrefix + ApiVersionLatest;
+        private const string ApiVersionQueryPrefix = "api-version=";
+        private const string ApiVersionGA = "2016-02-03";
+        private const string ApiVersionDefault = "2019-03-30";
```

Comment:
Do we still need two?

Reply:
Need two what?

I chose not to do the work to make sure we could get rid of the GA version for now, but I put it on a personal task list to look at later. It seems we just never did the work to move 3 calls from this older api-version.

Reply:
okay